src/Umbraco.Cms.Api.Management/Services/Signs/ISignProvider.cs

```diff
@@ -0,0 +1,31 @@
+using Umbraco.Cms.Api.Management.ViewModels.Tree;
+using Umbraco.Cms.Core.Models.Entities;
+
+namespace Umbraco.Cms.Api.Management.Services.Signs;
+
+/// <summary>
+/// Defines operation for the provision of presentation signs for tree and collection nodes.
+/// </summary>
+public interface ISignProvider
+{
+    /// <summary>
+    /// Prefix of each sign alias.
+    /// </summary>
+    const string Prefix = "Umb.";
```

Comment:
This is fine here, but maybe for consistency it should move into `Constants-Conventions.cs`